server/tidb_serial_test.go

```diff
@@ -363,6 +363,7 @@ func TestPrepareCount(t *testing.T) {
 	err = qctx.GetStatement(stmt.ID()).Close()
 	require.NoError(t, err)
 	require.Equal(t, prepareCnt, atomic.LoadInt64(&variable.PreparedStmtCount))
+	qctx.Close()
```

Comment:
```suggestion
	require.NoError(t, qctx.Close())
```

Reply:
%s/rows.Close()/require.NoError(t, rows.Close())/g in tidb_serial_test.go